src/component/requests/request.js

```diff
@@ -0,0 +1,113 @@
+import React from 'react';
+import FullUrl from './full-url.js';
+import RequestLabels from './request-labels.js';
+import ContextMenu from '../context-menu/context-menu.js';
+
+const {func, object, number, bool} = React.PropTypes;
+
+export default class Request extends React.Component {
+
+  shouldComponentUpdate(nextProps) {
+    return this.props.request.id !== nextProps.request.id ||
+      this.props.positionTop !== nextProps.positionTop ||
+      this.props.isActive !== nextProps.isActive ||
+      this.props.isContextMenu !== nextProps.isContextMenu ||
+      this.done !== nextProps.request.done;
```

Comment:
I know that `shouldComponentUpdate` was there before, but is it _really_ needed? Why only on `Request`?